staging/src/k8s.io/cli-runtime/pkg/resource/builder.go

```diff
@@ -710,6 +712,12 @@ func (b *Builder) SingleResourceType() *Builder {
 	return b
 }
 
+// NoSelector indicates that subcommand have no flag --all and --selector
+func (b *Builder) NoSelector() *Builder {
```

Comment:
Don't add unnecessary information to this interface, just change the message below.

Reply:
ok. fixed.

Reply:
This comment still holds, by not adding I've meant not adding new methods here.

Reply:
push again.